app/scripts/controllers/meeting.status.ctrl.js

```diff
@@ -58,17 +58,12 @@ angular.module('dashboard')
                             return angular.equals(item, $scope.selected);
```

Comment:
After $scope.selected is removed, whole use-case for `isSelected` disappears? It's dead code and should be removed from .js and .html?